api/client/service/update.go

```diff
@@ -122,8 +117,8 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 	updateString(flagName, &spec.Name)
 	updateLabels(flags, &spec.Labels)
 	updateString("image", &cspec.Image)
-	updateSlice("command", &cspec.Command)
-	updateSlice("arg", &cspec.Command)
+	updateStringToSlice(flags, "command", &cspec.Command)
```

Comment:
I'm just following what swarmkit supports. I think it would make more sense for swarmkit to return to the entrypoint/command behaviour we have with `docker run`.

I don't want to introduce a third way of resolving these here, and attempting to restore the old behaviour in the cli doesn't feel right either.